theme/src/components/auth-header.js

```diff
@@ -0,0 +1,28 @@
+import jwt from "jsonwebtoken"
```

Comment:
Why you have moved this file to here? Just change import AuthHeader from "../../../../../src/api/server/lib/auth-header" -> import AuthHeader from "../../../../.src/api/server/lib/auth-header" and it will work. Sensible data encoding and decoding should not be stay in theme folders.